lib/cisco_node_utils/cmd_ref/tacacs_server_host.yaml

```diff
@@ -1,37 +1,66 @@
 # tacacs_server_host
 ---
-_exclude: [ios_xr]
+_template:
+  nexus:
+    get_command: "show run tacacs all"
+  ios_xr:
+    get_command: "show running-config tacacs-server"
 
 encryption:
-  set_value: "%s tacacs-server host %s key %s %s"
+  nexus:
+    set_value: "<state> tacacs-server host <ip> key <key>"
+  ios_xr:
+    context: ["tacacs-server host <ip> port <port>"]
+    get_value: '/key \d+\s+(\S+)/'
```

Comment:
Don't need a ``get_value`` here, since we are only ever using this entry from ``set_config``